File: src/Symbolizer/gunction.c
#include <stdio.h>
#include <string.h>

struct Student {
	int i; 
	char name[100];
};

int gunction(struct Student* sptr, int d) {
	sptr->i = d / 100;
	strcpy(sptr->name, "Hello");
	if (sptr->i > 10) {
		sptr->i = 300;
	}	

	if (sptr->i > 100) {
		printf("%d\n", sptr->i);
	}
	return 0;
}

File: src/Symbolizer/testcase/Rust/gunction.rs
struct Student {
    i: i32,
//    name: String,
    j: i32,
}

fn gunction(sptr: &mut Student, d: i32) -> i32 {
    sptr.i = d / 100;
    // sptr.name = String::from("Hello");

    if sptr.i > 10 {
        sptr.i = 300;
    }

    if sptr.i > 100 {
        println!("{}", sptr.i);
    }
    
    sptr.j = sptr.j + sptr.j;
    0
}

fn main() {
    let mut student = Student { i: 0, j: 5 };
    gunction(&mut student, 2500);
}
